configure.ac

```diff
@@ -420,6 +420,9 @@ fi
 AC_MSG_RESULT([${enable_dbengine}])
 AM_CONDITIONAL([ENABLE_DBENGINE], [test "${enable_dbengine}" = "yes"])
 
+AC_DEFINE([ENABLE_METADATALOG], [1], [netdata metadata log])
```

Comment:
Why do we have this? Won't it be switched on all the time...

Reply:
Yes, it's a leftover, it should go away.

Reply:
Wait, it's not there, it was the initial commit. You can look at the final version to make sure.

Reply:
Branch mess. Let me rebase first.